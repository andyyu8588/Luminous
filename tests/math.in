import Math

math = Math();

print(math.abs(0));
print(math.abs(1));
print(math.abs(-1));

print(math.factorial(0));
print(math.factorial(1));
print(math.factorial(5));

print(math.isEven(0));
print(math.isEven(1000));
print(math.isEven(859));

print(math.isOdd(0));
print(math.isOdd(1000));
print(math.isOdd(859));

print(math.power(0, 1));
print(math.power(2, 3));
print(math.power(10, 5));