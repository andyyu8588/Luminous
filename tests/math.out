0
1
1
1
1
120
true
true
false
false
false
true
0
8
100000
